### ISSUE TITLE
Accessibility improovements

### ISSUE THREAD

**Issue by DraganRatkovich** [telegramdesktop/tdesktop#17030]
### Is your feature request related to a problem?

First, if such an issue is already open, please, close it.
Blind people cannot use the Telegram Desktop client due to the lack of accessibility support, especially screen reader support, not available in the Telegram desktop client.
Windows 10 has a client called "Unigram" that works very well with screen readers, but the said client only works on Windows 10, so it's not possible to use it on lower windows like Windows 8.1.
Please add screen reader support to the Telegram Desktop client so that we blind users can use it in the lower windows without any problems.
The only way we can use Telegram in the lower windows is by using the web version of it.
Hopefully the developers will take this issue into account.

### Describe the solution you'd like

Add support for screen readers like NVDA, Jaws, etc. This will help blind people to use Telegram client without problems.

### Describe alternatives you've considered

Currently, the alternative is to use the web version of Telegram

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
